.github/workflows/devcontainer-docker-image.yml

```
@@ -22,7 +22,7 @@ jobs:
       uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
     - name: Setup Docker buildx
-      uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6
+      uses: docker/setup-buildx-action@5e716dcfd653738c2d1db099bfba194a84158be4
```

I think we can pin to regular version numbers
```suggestion
      uses: docker/setup-buildx-action@v2.4.0
```